x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/content_types/keyword_content.tsx

```diff
@@ -50,14 +50,14 @@ export const KeywordContent: FC<FieldDataCardProps> = ({ config }) => {
         />
       </div>
 
-      <EuiSpacer size="m" />
+      <EuiSpacer size="l" />
 
       <div>
         <FormattedMessage
           id="xpack.ml.fieldDataCard.cardKeyword.topValuesLabel"
           defaultMessage="top values"
```

Comment:
I think the spacers either side of `top values` are too big. We could save vertical height here:

![image](https://user-images.githubusercontent.com/7405507/72254550-47272580-35fc-11ea-943d-2a0f882d6f0e.png)

Reply:
cc @mdefazio